fru.c

```diff
@@ -69,7 +69,7 @@ static inline uint8_t fru_field_copy(void *dest, const fru_field_t *fieldp)
  *
  */
 static
-uint8_t fru_get_typelen(int len,             /**< [in] Length of the data or LEN_AUTO for pure text zero-terminated data */
+uint8_t fru_get_typelen(int len,             /**< [in] Length of the data, LEN_AUTO for pure text zero-terminated data or one of LEN_BCDPLUS, LEN_6BITASCII, LEN_TEXT for explicit text type */
```

Comment:
Let's wrap this long line please to fit into 130 characters max.
I suggest wrapping it at `or one...` and aligning that part the the start of `Length of the data...`:

```
/**< [in] Length of the data...
          or one of...
```

Reply:
Split it.